torch/distributed/distributed_c10d.py

```diff
@@ -4297,6 +4298,175 @@ def _get_backend_from_str(backend: Optional[str] = None) -> Backend:
     return Backend(backend)
 
 
+@_time_logger
+def split_group(
+    parent_pg=None,
+    split_ranks=None,
+    timeout=None,
+    pg_options=None,
+    group_desc=None,
+):
+    """
+    Create a new distributed group splitted from the given parent group.
+
+    users of this API must gurantee that all ranks in the parent group enter this API call.
+    And the split of the group is the same accross the ranks.
+
+    Args:
+        parent_pg (ProcessGroup, optional): The parent process group. If None,
+            the default process group will be used. Users need to gurantee that 
+            the parent group is fully initialized (e.g, communicators are initialized)
+        split_ranks (list[int]): the split ranks, which is a list of list of ranks.
+            Users need to make sure the validity of the split rannks such that one 
+            split (represented by one inner list of ints) does not overlap with any other split.
+            note, the ranks in each split is the group rank in the parent pg.
+        timeout (timedelta, optional): see `init_process_group` for details and default value.    
+        group_desc (str, optional): a string to describe the process group.
+
+    Returns:
+        A handle of distributed group that can be given to collective calls or
```

Comment:
s/handle/class`torch.distributed.ProcessGroup`

(or whatever the right way to refer to a class in docstrings is)